### PR TITLE
[Data Viz] Implement Scatter Plot Chart type

### PR DESCRIPTION
**Pull request by ajpal**
# Description
Implements the scatter plot view:
image

## Links

- [spec](https://docs.google.com/document/d/1s1CQ4SSrpwJtLwI5gJoFzJEbcEXZHQIQ6ikAxB9H3j4/edit)
- [jira](https://codedotorg.atlassian.net/browse/STAR-753)

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by islemaster — COMMENTED** [review]
Unit test?

**Review by jmkulwik — APPROVED** [review]
LGTM!

File: apps/test/unit/storage/dataBrowser/DataVisualizerTest.js [hunk 1]
```diff
@@ -27,6 +28,52 @@ describe('DataVisualizer', () => {
...
+        ]
+      };
+      wrapper = shallow(<DataVisualizer {...STARTING_PROPS} />);
+      wrapper.instance().handleOpen();
+      var chartArea = document.createElement('div');
+      chartArea.setAttribute('id', 'chart-area');
+      document.body.appendChild(chartArea);
+    });
+
+    afterEach(() => spy.restore());
```
Comment (islemaster) [inline apps/test/unit/storage/dataBrowser/DataVisualizerTest.js]: Please update this test to clean up the DOM.

**Review by islemaster — CHANGES_REQUESTED** [review]

**Review by islemaster — APPROVED** [review]
:tada:

**Pull request merged**